react/features/file-sharing/middleware.web.ts

```diff
@@ -23,22 +26,66 @@ import { downloadFile } from './utils';
  */
 StateListenerRegistry.register(
     state => state['features/base/conference'].conference,
-    (conference, { dispatch }, previousConference) => {
+    (conference, { dispatch, getState }, previousConference) => {
         if (conference && !previousConference) {
             conference.on(JitsiConferenceEvents.FILE_SHARING_FILE_ADDED, (file: IFileMetadata) => {
                 dispatch(addFile(file));
+
+                const state = getState();
+                const localParticipant = getLocalParticipant(state);
+                const isRemoteFile = file.authorParticipantId !== localParticipant?.id;
+
+                if (isRemoteFile) {
+                    const { isOpen, focusedTab } = state['features/chat'];
+                    const isFileSharingTabVisible = isOpen && focusedTab === ChatTabs.FILE_SHARING;
+
+                    if (!isFileSharingTabVisible) {
+                        dispatch({
+                            type: INCREMENT_UNREAD_FILES_COUNT
+                        });
+                    }
+
+                    dispatch(showNotification({
```

Comment:
Do we show chat notifications when the chat pane is open? I think not, should we do the same here?